components/sealion-cubesat/components/sealion-power-module/components/battery-pack/package.json

```diff
@@ -0,0 +1,9 @@
+{
+    "name": "battery-pack",
+    "version": "0.1.0",
+    "description": "Physically holds 10 onboard batteries in 5S2P configuration to deliver electrical power to EPS Board.",
```

Comment:
The description should be a human-readable version of the `name`

Reply:
Though, I also like having this verbiage as a sort of annotation for the component, so let's copy this verbiage to a `docs` field; i.e., create a line under `description` called `docs` with the above verbiage, i.e.,
```
"docs": "Physically holds 10 onboard batteries in 5S2P configuration to deliver electrical power to EPS Board.",
```
I know `docs` isn't in the official DOF spec, but it's something that's come up as a worth while keyword to be adopted

Reply:
same goes for the 11 other `package.json` files

Reply:
On second thought, these notes should go in the `notes` field in the `parts.yaml`